[DOCS-14454] Add description front matter to metrics guide pages (#36885)
DataDog/documentation: content/en/metrics/guide/interpolation-the-fill-modifier-explained.md

---
title: Interpolation and the Fill Modifier

> description: "Understand how interpolation and the fill modifier align metric series so you can aggregate across sources."

Reviewer: ```suggestion
description: "Understand how interpolation and the fill modifier help you aggregate across sources by aligning metric series."
```
I wasn't familiar with interpolation and the fill modifier beforehand, so I had some trouble parsing out where the unfamiliar noun phrase ended. I'm not sure if this slight rewording helps; feel free to ignore if you prefer your current version!
